geode-core/src/main/java/org/apache/geode/cache/RegionService.java

```diff
@@ -74,7 +74,10 @@
    * Returns a factory that can create a {@link PdxInstance}.
    *
    * @param className the fully qualified class name that the PdxInstance will become when it is
-   *        fully deserialized.
+   *        fully deserialized unless {@link PdxInstanceFactory#neverDeserialize()} is called.
```

Comment:
It's better to document the use case when empty string will be used as `className`. Or give an example explaining why empty string is the `className`. It will provide more context. Since it is a special case.